styles/widgets/_classTree.scss

```diff
@@ -20,25 +20,107 @@
     border-top: 1px solid rgb(221, 221, 221);
   }
 
-  .jstree-icon.default-tree-icon::after {
+  &__elements {
+    margin-left: -20px;
+  }
+
+  .parent-tree-icon {
+      content: url("../images/tree-leaf-folder.png");
+      max-width: 19px;
+      max-height: 19px;
+      margin: -5px;
```

Comment:
I consider this styles as pretty unintuitive: why do you so many margin, padding or coordinate positioning (top, left) rules? It's really hard to understand all this interaction between e.g. `position: relative`, `float: left`, `width: fit-content`, `left: 30px`, `text-align: center`, `vertical-align: middle` and `line-height: 1`.

Reply:
Pavlov Dmitry asked me to specify the count of instances in the same style as the `.ontodia-badge` element (`...ontodia\styles\viewUtils\_badge.scss`)
All attributes are taken exactly from the element, nothing added from me.

Reply:
Maybe it would be a better idea to reuse `.ontodia-badge` class instead of duplicatig the style rules? We should probably treat these styles as legacy (because they're partially reused from Twitter Bootstrap).

Reply:
Done.